### DISCUSSION TITLE
Feature suggestions for Visual Studio 2022 Co Pilot Chat

### DISCUSSION THREAD

**Discussion by grahammcallister** [community/community#57426] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I'm not sure where/how to give feedback on feature suggestions or experience on the Visual Studio 2022 Co Pilot Chat feature.

Some suggestions / requests from my side:

- A way to save my previous conversations 
- A way to "pull" parts of various conversations into other conversations, as they become relevant while I work
- A way to add a colleague into the chat so we can all work together with Co-Pilot
- A clearer way of indicating which files are "in scope" for Co Pilot (apparently it's any open file editor windows but is this really true?)
- Seems others have mentioned but the input control for the chat doesn't seem to support pasting multiple lines of code that I want to use to clarify my questions.

I'm going to list further suggestions here going forward. Or is there a better way / place to interact with the team?

**Comment by grahammcallister** [comment]
![image](https://github.com/community/community/assets/370605/0acfb62b-627f-4617-8dba-08f8efe8950f)

Note this image which shows a CoPilot response to the question "Which files can you see open in my editor currently". The response seems unintuitive to me, since I expected CoPilot to have a working knowledge of the files I am working on?

**Comment by perk-bevis** [comment]
great
